src/sparql/ItemValueBuilder.ts

```diff
@@ -0,0 +1,158 @@
+import PropertyValueRelation from '@/data-model/PropertyValueRelation';
+import { Condition } from '@/sparql/QueryRepresentation';
+import rdfNamespaces from '@/sparql/rdfNamespaces';
+import SyntaxBuilder from '@/sparql/SyntaxBuilder';
+import TripleBuilder from '@/sparql/TripleBuilder';
+import ValuePatternBuilder from '@/sparql/ValuePatternBuilder';
+import { MinusPattern, Pattern, PropertyPath, Term } from 'sparqljs';
+
+export default class ItemValueBuilder implements ValuePatternBuilder {
+	private readonly tripleBuilder: TripleBuilder;
+
+	public constructor() {
+		this.tripleBuilder = new TripleBuilder();
+	}
+
+	public buildValuePatternFromCondition( condition: Condition, conditionIndex: number ): Pattern[] {
+		const {
+			propertyId,
+			referenceRelation,
+			propertyValueRelation,
+			datatype,
+			value,
+			negate,
+			subclasses,
+		} = condition;
+
+		if ( datatype !== 'wikibase-item' ) {
+			throw new Error( 'Expected datatype wikibase-item, got: ' + datatype );
+		}
+		if ( typeof value !== 'string' ) {
+			throw new Error( 'Unexpected wikibase-item value type: ' + typeof value );
+		}
+		let patterns: Pattern[] = [];
+
+		const statementVariable = SyntaxBuilder.buildVariableTermFromName( 'statement' + conditionIndex );
```

Comment:
Same suggestion from the previous PR. Move SyntaxBuilder to class dependencies to make it clear (and maybe easier to write unit test for later)

Reply:
Mh, his shouldn't be mocked by unit tests, it is part of this unit on a fundamental level. But making the dependency a bit more clear can't hurt.

Reply:
Done.